orgs/prerelease.json

```diff
@@ -0,0 +1,19 @@
+{
+  "orgName": "Outbound Funds - Feature Test Org",
+  "edition": "Developer",
+  "instance": "cs46",
+  "settings": {
+    "chatterSettings": {
+      "enableChatter": true
+    },
+    "apexSettings": {
+      "enableDisableParallelApexTesting": true
```

Comment:
Is this one necessary?

Reply:
Probably not, but it's on in all of the existing org files. I'd like to experiment separately with turning that off and converting to SFDX format.